addons/mail/models/mail_thread.py

```diff
@@ -3247,9 +3248,10 @@ def _notify_thread(self, message, msg_vals=False, **kwargs):
             # generate immediately the <mail.notification>
             # and send the <mail.mail>, <mail.push> and the <bus.bus> notifications
             self._notify_thread_by_inbox(message, recipients_data, msg_vals=msg_vals, **kwargs)
-            self._notify_thread_by_email(message, recipients_data, msg_vals=msg_vals, **kwargs)
+            if thread_model != "discuss.channel":
+                self._notify_thread_by_email(message, recipients_data, msg_vals=msg_vals, **kwargs)
```

Comment:
As this change is for Discuss Channels only it should be done in the Discuss folder itself.